Starting logfile at 18:28:10.555[06-15-2015 18:28:10.555] Completed args reading
[06-15-2015 18:28:14.811] From Client: @Sure
[06-15-2015 18:28:14.811] Loading driver...
[06-15-2015 18:28:14.875] Driver loaded!
[06-15-2015 18:28:14.875] Getting connection
[06-15-2015 18:28:15.10] Connected to database
[06-15-2015 18:28:15.11] executeCommand invoked
[06-15-2015 18:28:15.11] stmt initialized & declared
[06-15-2015 18:28:15.11] rs declared
[06-15-2015 18:28:15.11] Query: SELECT yesno_validations FROM yesno_answers WHERE yesno_id = 'Sure';
[06-15-2015 18:28:15.13] Query executed
[06-15-2015 18:28:15.13] Closing connection
[06-15-2015 18:28:15.13] Result of findData(): #
[06-15-2015 18:28:15.14] executeCommand invoked
[06-15-2015 18:28:15.14] stmt initialized & declared
[06-15-2015 18:28:15.14] rs declared
[06-15-2015 18:28:15.14] Query: SELECT yesno_tf FROM yesno_answers WHERE yesno_id = 'Sure';
[06-15-2015 18:28:15.15] Query executed
[06-15-2015 18:28:15.15] Closing connection
[06-15-2015 18:28:15.15] Result of findData(): #
[06-15-2015 18:28:15.15] Output: %
[06-15-2015 18:28:15.15] Sending: %
[06-15-2015 18:28:15.16] Message sent
[06-15-2015 18:28:15.16] Closing connections
[06-15-2015 18:28:15.17] Connections closed
[06-15-2015 18:28:22.922] From Client: *Sure
[06-15-2015 18:28:22.922] Loading driver...
[06-15-2015 18:28:22.922] Driver loaded!
[06-15-2015 18:28:22.922] Getting connection
[06-15-2015 18:28:22.928] Connected to database
[06-15-2015 18:28:22.929] createRow invoked
[06-15-2015 18:28:22.929] stmt initialized & declared
[06-15-2015 18:28:22.929] Command: INSERT INTO yesno_answers VALUES (Sure, 1, 1);
[06-15-2015 18:28:22.933] Exception caused by sending a command (createRow)
[06-15-2015 18:28:22.933] Closing connection
[06-15-2015 18:28:22.933] Output: $
[06-15-2015 18:28:22.933] Sending: $
[06-15-2015 18:28:22.934] Message sent
[06-15-2015 18:28:22.934] Closing connections
[06-15-2015 18:28:22.935] Connections closed
